index.html

```diff
@@ -3163,7 +3163,30 @@ <h2>Syntaxes</h2>
 validation, or comparison. This specification makes no requirements for support
 of any specific serialization format.
       </p>
-
+      <p>
+The expected arity of the property values in this specification, and the
+resulting datatype which must be used to hold those values, can vary depending
+on the property. If present, the following properties MUST be represented as a
```

Comment:
This MUST is effectively stated throughout the specification and so re-stating it is typically frowned upon. You don't want to restate the same normative requirement in two different ways because it can lead to confusion when text is edited elsewhere in the specification.